build.sbt

```diff
@@ -54,10 +50,15 @@ lazy val microservice = Project(appName, file("."))
     Assets / pipelineStages := Seq(gzip),
     unitTestSettings,
     acceptanceTestSettings,
-    integrationTestSettings,
     resolvers += Resolver.jcenterRepo
   )
 
+lazy val it = project
+  .enablePlugins(PlayScala)
+  .dependsOn(microservice % "test->test")
+  .settings(DefaultBuildSettings.itSettings())
+  .settings(sharedSettings)
```

Comment:
some of this was new to me, but https://github.com/hmrc/sbt-settings/tree/435efe9344b8c6fa3a13e00879d1d9f9e462115b  explains it really well 🙌